stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/extensions/filter.py

```diff
@@ -171,6 +183,16 @@ def to_es(self):
                     "must_not": [{"term": {to_es(self.args[0]): to_es(self.args[1])}}]
                 }
             }
+        elif self.op == AdvancedComparisonOp.like:
+            return {
+                "wildcard": {
+                    to_es(self.args[0]): {
+                        "value": cql2_like_to_es(str(to_es(self.args[1]))),
+                        "boost": 1.0,
+                        "case_insensitive": "true",
```

Comment:
I think this should be case sensitive

Reply:
Do you mean that `case_insensitive` should be set to `false`?

Reply:
yes, since AFAIK LIKE is a case sensitive operation (e.g., `b%` matches `ba` but not `Ba`

Reply:
Got it. Updated in [e7330dc](https://github.com/stac-utils/stac-fastapi-elasticsearch/pull/178/commits/e7330dc7ab9b18d1c92b0075e02520c9afdb97e6)